gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/CockpitAuthenticationFilter.java

```diff
@@ -154,41 +154,45 @@ public void doFilter(jakarta.servlet.ServletRequest request, jakarta.servlet.Ser
     }
 
     private boolean enabled() {
-        return configuration.getProperty("cockpit.enabled", Boolean.class, false);
+        return getProperty("cockpit.enabled", "cloud.enabled", Boolean.class, false);
```

Comment:
**question**: do we have to test the property starting with cockpit first ? :thinking:  I was thinking the cloud one should be tested first and then fallback to the cockpit one at it is legacy. (this comment is for all the changes related to this getProperty method)

Reply:
Hi @leleueri 
We don't wish to introduce any breaking change which is why cockpit configuration takes precedence over cloud conf.

Reply:
ok, as there is a fallback IMO this is not a BC but lets keep it like this.

Reply:
Not really, depending of the installation type. 
If a user have `cockpit.enabled = true` and process to upgrade, the user will have both `cockpit.enabled = true` and `cloud.enbaled = false`